### DISCUSSION TITLE
can I disable walletconnect?

### DISCUSSION THREAD

**Discussion by guotie** [family/connectkit#308] [Q&A] [answered]
I don't want use walletconnect, how to disable it?

**Comment by lochie** [comment, marked as answer]
> [!WARNING]  
> Disabling WalletConnect will remove core functionality from ConnectKit and cause unexpected behaviour, especially for users on mobile devices.

You can extend and customise the connectors before pushing your config to the wagmi createConfig

```ts
const ckConfig = getDefaultConfig({
  ... // your config
})

ckConfig.connectors = [
  ...ckConfig.connectors.filter( ... )
];

const config = createConfig(ckConfig); // wagmi's createConfig
```
